main.go

```diff
@@ -164,6 +167,9 @@ func CrawlPaths(paths []string) chan string {
 				continue
 			}
 			filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+				if !*vendor && (strings.HasPrefix(path, "vendor/") || strings.Contains(path, "/vendor/")) {
```

Comment:
`filepath.Walk` deals with system filepaths, and the separator is not guaranteed to be `/` on every OS. As a result, checking if `strings.HasPrefix(path, "vendor/")` or `strings.Contains(path, "/vendor/") is not correct.